ui/app/AppLayouts/Profile/helpers/ProfileShowcaseDirtyState.qml

```diff
@@ -16,9 +16,18 @@ import utils 1.0
   * position, second one containing hidden items.
   */
 QObject {
+    id: root
+
     property alias sourceModel: joined.leftModel
     property alias showcaseModel: joined.rightModel
 
+    /**
+      * True if the showcase model is in single model mode, i.e. the showcase
+      * model is part of the source model. False if the showcase model is a
+      * separate model.
+      */
+    property bool singleModelMode: !joined.rightModel
```

Comment:
readonly?

Reply:
no, it's an input property. This is just the default property